### ISSUE TITLE
Can't use pre-trained data frozen_inference_graph.pbsegmentaa, ab, ac and ad

### ISSUE THREAD

**Issue by yh570** [Bellspringsteen/OurCamera#11]
Hi,

Thank you for your help on the previous issue. I successfully go through all the steps, but the output images don't contain any identified objects. To figure out the problem I'm trying to use your pre-trained data but analyzeimages.py can't read them because it's separated as frozen_inference_graph.pbsegmentaa, frozen_inference_graph.pbsegmentab, frozen_inference_graph.pbsegmentac and frozen_inference_graph.pbsegmentad. Would you please let me know how to use them with analyzeimages.py?

Edit: I just found the detection scores are pretty low with my training data which has more than 20,000 steps. So it looks like my training has some problem. I used the sample images in this git. Any comments about the problem? 

Thank you.

**Comment by Bellspringsteen** [comment]
Hello,

I split apart the model because Github has a file size limit. You can concatentate the files back together, for instance something like cat frozen_inference_graph.pbsegment?? > frozen_inference_graph.pb

What was your Total Loss value after 20,000 steps? Were you running tensorboard during the training?

**Comment by yh570** [comment]
Thank you for your reply, attached is my loss graph and evaluation images. I ran the tensorboard during all my training. The loss value is pretty low but the diagram looks strange. The evaluation images detect everything as the bicycle. I just tried your pre-trained model and the result is pretty good.

<img width="787" alt="screen shot 2018-03-29 at 11 25 57 pm" src="https://user-images.githubusercontent.com/16182315/38123795-144d7290-33ab-11e8-814f-f1c2e29149a9.png">
<img width="823" alt="screen shot 2018-03-29 at 11 26 26 pm" src="https://user-images.githubusercontent.com/16182315/38123805-1e2f8ea6-33ab-11e8-8a25-119d90f06d0a.png">

**Comment by Bellspringsteen** [comment]
Those images are from step 0. Tensorboard should rerun them for further steps. I dont know why you arent seeing eval images for later steps. You could try just exporting a frozen version of your model and running the analyzeimages to see the results.

I am glad you are seeing good results with the frozen model I posted.
